homeassistant/components/mqtt/number.py

```diff
@@ -16,6 +16,7 @@
     NumberMode,
     RestoreNumber,
 )
+from homeassistant.components.sensor.recorder import EQUIVALENT_UNITS
```

Comment:
This has other translations than between the different encodings of µ. I think it's weird to add those in this PR.

Reply:
Replaced with `AMBIGUOUS_UNITS`